### ISSUE TITLE
'Only open folder notes through the name' Setting Causes Inconsistent Styling

### ISSUE THREAD

**Issue by mackereldev** [LostPaul/obsidian-folder-notes#171]
I've noticed that when the setting '_Only open folder notes through the name_' is disabled, it causes the folder collapse icon to shift ~~and overflow ellipses on file explorer items to no longer appear~~. This is inconsistent with when the option is enabled, and with vanilla Obsidian. I think it's caused by flex elements not supporting ellipsis overflow on their text component, but I'm not certain.

I've created a proposed fix [here](https://github.com/mackereldev/obsidian-folder-notes) by I don't fully understand the implications of changing these styling rules. Have I overlooked something or am I clear to make a PR?

Edit: reverted changes made to the folder collapse icon because that issue was caused by a conflict with [Relay](https://github.com/no-instructions/relay).

**Comment by Jon-dog** [comment]
Yeah I've also noticed this recently. Should this add-on be touching the styling of those elements?

The only functions I see that would get involved is the "hide collapse icon" option and the folder note overview

**Comment by LostPaul** [comment]
> Looks to be gone now, which is good.
> A minor issue from the style updates is that if I open a folder with only a folder note, a single pixel of the outline column shows.
> This seems to happen whether I have **Hide folder note** enabled or not.
> 
> Tried disabling every style add-on I had + going back to the default theme and it was still there.
> 
> ![Image](https://github.com/user-attachments/assets/bb7f3b07-67b8-40e0-8fa2-8a2e3bad6f9b)

Can you reproduce it in a sandbox vault?

**Comment by Jon-dog** [comment]
Created one from scratch and only installed Brat + Folder Note, and it still appears

I know it's really tiny, but in coloured dark themes, a bright pixel sticks out
![Image](https://github.com/user-attachments/assets/7ef7e6d5-4b2e-41ae-88c3-14f6ae0d498a)

If you highlight the note folder, you can see that the folder below also moves a pixel down
https://github.com/user-attachments/assets/62ddba1c-890b-4a8f-b724-3c7bdce0694d

**Comment by LostPaul** [comment]
Should be fixed now, was a bit more complicated than I thought.

**Comment by Jon-dog** [comment]
Looks good to me now, I can't see any stray pixels
